CSS/style.css

```diff
@@ -4,30 +4,79 @@
 
 @import url("./Components/index.css");
 
+/* Docs- Getting started CSS */
+.gap {
+  margin-bottom: 2rem;
+}
+.list-heading {
+  padding: 1rem;
+  font-weight: 600;
+}
+.list-components {
+  padding-inline-start: 0;
+}
+.list-components a {
+  display: flex;
+  align-items: center;
+}
+.list-components li {
+  padding: 1rem;
+  font-weight: 500;
+  transition: all 0.3s ease-in-out;
+}
+.list-components li:hover {
+  background-color: var(--primary-color);
+  color: var(--off-white);
+  border-radius: 0 0 50% 0;
+  transition: all 0.3s ease-in-out;
+  font-weight: 600;
+}
+.active {
+  color: var(--primary-color);
+}
+.code {
+  color: var(--primary-color);
+}
+.next-pre-btns {
+  display: flex;
+  font-size: 1.2rem;
+  font-weight: 500;
+  justify-content: space-between;
+  margin-top: 3rem;
+}
+.mr-auto {
+  margin-right: auto;
+}
+.ml-auto {
```

Comment:
Just a small suggestion class name could mr-left-auto is easier to understand

Reply:
made the changes